htdocs/mrp/mo_list.php

```diff
@@ -207,6 +217,74 @@
 	$objectlabel = 'Mo';
 	$uploaddir = $conf->mrp->dir_output;
 	include DOL_DOCUMENT_ROOT.'/core/actions_massactions.inc.php';
+	$objMo = new Mo($db);
+
+	if ($action == 'confirm_cancel' && $confirm == 'yes') {
```

Comment:
There is a CI error here becasue you executed an action without doing any test on permission.
Try to replace with 

if ($action == 'confirm_cancel' && $confirm == 'yes' && $permissiontoadd